frontend/integration-tests/views/environment.view.ts

```diff
@@ -8,6 +8,12 @@ export const rowsValue = $('[placeholder="value"]');
 export const deleteBtn = $$('.pairs-list__delete-icon').first();
 export const saveBtn = element(by.cssContainingText('.btn.btn-primary', 'Save'));
 
+export const prefix = $$('input:only-child').last();
```

Comment:
It would be better to add a `data-test-id` attributes to the elements in the form. This selector is fragile. We're starting to use `data-test-*` more in console.

The following doc is for cypress not protractor, but it sums it up pretty well: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements